### DISCUSSION TITLE
How to disable highlighting for a specific plugin?

### DISCUSSION THREAD

**Discussion by satoshiyamamoto** [projekt0n/github-nvim-theme#284] [Q&A] [answered]
Hi, I like this theme and use it all the time.

I have a question on plugin support, is it possible to disable highlighting only for specified plugins (e.g. nvim-notify)?

**Comment by ful1e5** [comment, marked as answer]
@satoshiyamamoto To disable the plugin highlight, set the [module name](https://github.com/projekt0n/github-nvim-theme/tree/main/lua/github-theme/group/modules) to `false` in the `options` sub-table. 

Once you do that, you'll need to manually recompile the theme using `:GithubThemeCompile` because the auto re-compiling process is currently not working correctly.

```lua
require('github-theme').setup({
  options = {
    modules = {
      notify = false,
    },
    -- ...
  },
})

vim.cmd('colorscheme github_dark')
```

**Reply by satoshiyamamoto** [reply]
It worked as expected with the `options.modules` configuration.
I had missed the description in the Modules section of [README.md](https://github.com/projekt0n/github-nvim-theme#modules).
Thank you very much!
